src/wp-admin/includes/ajax-actions.php

```diff
@@ -5566,7 +5566,7 @@ function wp_ajax_send_password_reset() {
 	$user    = get_userdata( $user_id );
 	$results = retrieve_password( $user->user_login );
 
-	if ( true === $results ) {
+	if ( ! is_wp_error( $results ) ) {
```

Comment:
Both are same if not error then it's `true`. Why this change is needed?

Reply:
Looks like a change for consistency. Technically-speaking, I'd tend to favour "If true then success", as anything other than true should be ignored. While it sounds the same, if `retrieve_password()` was ever broken and returned `false`, a success message would be sent.